src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/service/cessations/BeneficialOwnerCessationService.java

```diff
@@ -0,0 +1,76 @@
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.BeneficialOwnerCorporateDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.BeneficialOwnerGovernmentOrPublicAuthorityDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.BeneficialOwnerIndividualDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.cessations.*;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.types.PersonName;
+
+@Service
+public class BeneficialOwnerCessationService {
+  private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
+
+  public List<Cessation> beneficialOwnerCessations(OverseasEntitySubmissionDto updateSubmission) {
+    overseasEntitySubmissionDto = updateSubmission;
+    List<Cessation> cessations = new ArrayList<>();
+    cessations.addAll(returnIndividualBeneficialOwners());
+    cessations.addAll(returnCorporateEntityBeneficialOwnerCessations());
+    cessations.addAll(returnLegalPersonBeneficialOwners());
+    return cessations;
+  }
+
+  private List<IndividualBeneficialOwnerCessation> returnIndividualBeneficialOwners() {
+    var returned = overseasEntitySubmissionDto.getBeneficialOwnersIndividual();
+    return returned.stream()
+        .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getCeasedDate() != null))
+        .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getChipsReference() != null))
+        .map(this::returnIndividualBeneficialOwnerCessation)
+        .collect(Collectors.toList());
+  }
+
+  private List<LegalPersonBeneficialOwnerCessation> returnLegalPersonBeneficialOwners() {
+    var returned = overseasEntitySubmissionDto.getBeneficialOwnersGovernmentOrPublicAuthority();
+    return returned.stream()
+            .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getCeasedDate() != null))
+            .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getChipsReference() != null))
+            .map(this::returnLegalPersonBeneficialOwnerCessation)
+            .collect(Collectors.toList());
+  }
+
+  private List<CorporateEntityBeneficialOwnerCessation>
+      returnCorporateEntityBeneficialOwnerCessations() {
+    var returned = overseasEntitySubmissionDto.getBeneficialOwnersCorporate();
+    return returned.stream()
+            .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getCeasedDate() != null))
+            .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getChipsReference() != null))
+            .map(this::returnCorporateEntityBeneficialOwnerCessation)
+            .collect(Collectors.toList());
+  }
+
+  public IndividualBeneficialOwnerCessation returnIndividualBeneficialOwnerCessation(
+      BeneficialOwnerIndividualDto bo) {
+    return new IndividualBeneficialOwnerCessation(
+            bo.getChipsReference(),
```

Comment:
The appointment ids need to come from the collated data and are in the private part. Here it's using the ch_references which are the hash of the id.